### PR TITLE
sol

### PR DESCRIPTION
**Pull request by vladyslavchichkan**

### CODE REVIEW

File: cinema/urls.py [hunk 1]
```diff
@@ -1 +1,24 @@
...
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```
Comment (DevSpaciX) [inline cinema/urls.py]: When basepath is empty no need to include, instead use:
urlpatterns = router.urls

File: cinema/views.py [hunk 2]
```diff
@@ -1 +1,70 @@
...
+
+class MovieSessionViewSet(viewsets.ModelViewSet):
+    queryset = MovieSession.objects.all()
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return MovieSessionListSerializer
+        elif self.action == "retrieve":
+            return MovieSessionRetrieveSerializer
+        return MovieSessionSerializer
```
Comment (DevSpaciX) [inline cinema/views.py]: Could you just return self.serializer_class here ?

File: cinema/views.py [hunk 3]
```diff
@@ -1 +1,70 @@
...
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return MovieListSerializer
+        elif self.action == "retrieve":
+            return MovieRetrieveSerializer
+        return MovieSerializer
```
Comment (DevSpaciX) [inline cinema/views.py]: Could you just return self.serializer_class here ?

**Review by DevSpaciX — CHANGES_REQUESTED** [review]

File: cinema/views.py (hunk 3 above)
Comment (DevSpaciX) [reply cinema/views.py]: Oh , and delete pls db.sqlite3 file from PR

File: cinema/views.py (hunk 2 above)
Comment (Unlie9) [reply cinema/views.py:64]: please explain why i need to return self.serializer_class here

**Review by Arsen-hrynevych — APPROVED** [review]
Thanks for your effort

**Pull request closed without merging**